<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Widget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>531</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>401</width>
      <height>631</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="FileButton">
           <property name="text">
            <string>File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="GifButton">
           <property name="text">
            <string>Start GIF Recording</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ScrenshotButton">
           <property name="text">
            <string>Screncshoot</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="PathLabel">
         <property name="text">
          <string>File: </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="VertexLabel">
           <property name="text">
            <string>Vertex:0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="FacetesLabel">
           <property name="text">
            <string>Facetes:0</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="xTransitionLabel">
           <property name="text">
            <string>x Transition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="xTransitionSlider">
           <property name="minimum">
            <number>-100</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xTransitionSpinBox">
           <property name="minimum">
            <double>-99.989999999999995</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="yTransitionLabel">
           <property name="text">
            <string>y Transition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="yTransitionSlider">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yTransitionSpinBox">
           <property name="minimum">
            <double>-99.989999999999995</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="zTransitionLabel">
           <property name="text">
            <string>z Transition</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="zTransitionSlider">
           <property name="minimum">
            <number>-99</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zTransitionSpinBox">
           <property name="minimum">
            <double>-99.989999999999995</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="xRotationLabel">
           <property name="text">
            <string>x Rotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="xRotationSlider">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="xRotationSpinBox">
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="yRotationLabel">
           <property name="text">
            <string>y Rotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="yRotationSlider">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="yRotationSpinBox">
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="zRotationLabel">
           <property name="text">
            <string>z Rotation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="zRotationSlider">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zRotationSpinBox">
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="ScaleLabel">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="ScaleSlider">
           <property name="minimum">
            <number>-10</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="ScaleSpinBox">
           <property name="minimum">
            <double>-10.000000000000000</double>
           </property>
           <property name="maximum">
            <double>10.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="LineWidthLabel">
           <property name="text">
            <string>Line Width</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="LineWidthSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="PointSizeLabel">
           <property name="text">
            <string>Point Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="PointSizeSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Vertex View</string>
       </property>
       <widget class="QRadioButton" name="EmptyRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>32</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Empty</string>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="CircleRadioButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>32</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Circle</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="SquareRadioButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>32</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Square</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Line</string>
       </property>
       <widget class="QRadioButton" name="dottedRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>196</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Dotted line</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="SolidRadioButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>397</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Solid line</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>View</string>
       </property>
       <widget class="QRadioButton" name="perspectiveRadioButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>129</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Perspective</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="ParralelRadioButton">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>30</y>
          <width>195</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Parralel</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="FadeColorButton">
         <property name="text">
          <string>Edge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="VertexColorButton">
         <property name="text">
          <string>Vertex</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Background</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ModelColorButton">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">widget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
